04stable.autobuild

```diff
@@ -115,6 +115,8 @@ in_flavor 'master', 'stable' do
                         end
                     end
                 end
+            elsif shell.nil?
+                Autoproj.error("Unable to detect shell type. Please set the SHELL environment variable.")
```

Comment:
" or disable shell helpers altogether by setting the `shell_helpers` configuration variable to false"